lib/Minz/ExtensionManager.php

```diff
@@ -88,7 +111,9 @@ class Minz_ExtensionManager {
 	 * <name> must match with the entry point in metadata.json. This class must
 	 * inherit from Minz_Extension class.
 	 */
-	public static function init() {
+	public static function init(): void {
+		self::reset();
```

Comment:
During a cron job to refresh feeds, extensions enabled by a previous use risked being enabled for another user